### DISCUSSION TITLE
Development: where do CSS files go?

### DISCUSSION THREAD

**Discussion by daiyi** [Day8/re-frame-10x#294] [Q&A]
Originally, the LESS and CSS files were in `src/day8/re_frame/trace/css` and `src/day8/re_frame/trace/less`.

Then, they were both moved to `resources/day8/re_frame/trace` because it's more proper naming convention to places styles into the resources folder.

However, now the host project can't find the CSS, because we gave instructions to change the source-paths to look like this: 

```
:source-paths ["src" "../../src" "checkouts/re-frame-trace/src"]
```
and `resources` sits outside of this.

Which is a preferable fix?
1) adding "checkouts/re-frame-trace/resources" to `:source-paths`
2) moving the LESS and CSS files back into `src/day8/re_frame/trace`

**Actionable item**: update the README and/or file locations to reflect the decision made here!

**Comment by danielcompton** [comment]
1 is the better fix. `:source-paths` is only for Clojure files, not CSS/LESS.

**Comment by danielcompton** [comment]
I think the current behaviour is fine, I only need to add the src folder to my checkouts and the host project can still find/update new CSS.
